app/models/host/managed.rb

```diff
@@ -284,7 +284,8 @@ def handle_realm
   # returns the host correct disk layout, custom or common
   def diskLayout
     @host = self
-    pxe_render((disk.empty? ? ptable.layout : disk).gsub("\r",""))
+    template = disk.blank? ? ptable.layout : disk
```

Comment:
fixed via https://github.com/theforeman/foreman/pull/2524/files#diff-6632a0077c474a44423343eb597f9d72R158, I thought raising exception is actually better than rendering empty string as ptable which would cause an issue too late